django_client_framework/api/related_model_api.py

```diff
@@ -93,7 +98,10 @@ def post(self, request, *args, **kwargs):
         self.__check_write_perm_on_rel_objects(
             self.field_model.objects.filter(pk__in=self.__body_pk_ls)
         )
-        self.field_val.add(*self.__body_pk_ls)
+
+        selected = list(self.field_model.objects.filter(id__in=self.__body_pk_ls))
+        for selected_product in selected:
+            self.field_val.add(selected_product)
         return self.__return_get_result_if_permitted(request, *args, **kwargs)
```

Comment:
No need to use `list`. Simply using the `QuerySet` is enough. `list` forces `select *` to happen which is inefficient.